reviewdog.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/reviewdog/reviewdog/diff"
 )
 
+var X = 14
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported var X should have comment or be unexported